### PR TITLE
Added Basque language in settings and leave files

### PR DESCRIPTION
**Pull request by bipoza**

### CODE REVIEW

File: menu/translations/lxqt-settings_eu.directory [hunk 1]
```diff
@@ -0,0 +1 @@
+Name[eu]=LXQt-ren konfigurazioa
```
Comment (tsujan) [inline menu/translations/lxqt-settings_eu.directory:1]: Please add the missing newlines (an empty line at the end) to both files. GitHub complains about them.

Comment (bipoza) [reply menu/translations/lxqt-settings_eu.directory]: Done, I just added the missing lines.

**Comment by tsujan** [comment]
Thanks!

Please also squash your two commits into one for a correct commit history.

**Comment by tsujan** [comment]
@yan12125, @palinek, @luis-pereira,...

I couldn't find a pattern to explain why the "Squash and Merge" button sometimes work, sometimes doesn't. Do you know of a pattern?

**Comment by yan12125** [comment]
You may need to check if the button is really "Squash and Merge" when hitting it. Sometimes it jumps to merge or rebase even if I remember the last time I used squash. GitHub mysteries maybe.

**Comment by tsujan** [comment]
> You may need to check if the button is really "Squash and Merge" when hitting it.

Oh, I'm sure about that! For this PR, it worked. For https://github.com/lxqt/lxqt-themes/pull/18, it didn't and made a messy history.

We can't expect all contributors to know about squashing, especially beginners.

**Comment by yan12125** [comment]
No idea then :/

**Comment by palinek** [comment]
I always merged in command line. That way you can perfectly control, what you're doing.

**Comment by luis-pereira** [comment]
> I couldn't find a pattern to explain why the "Squash and Merge" button sometimes work, sometimes doesn't. Do you know of a pattern?

I usually do complex merges on the command line. With Github only `rebase and merge`.
@tsujan Are you referring to this repo or in general ?

**Comment by tsujan** [comment]
> @tsujan Are you referring to this repo or in general ?

It happened once with lxqt-panel's repository and when I pressed squashed-and-merge button to merge a contributor's work. 

I think it's a github bug: if it can't squash, it should show a message instead of merging when that button is pressed.

**Comment by luis-pereira** [comment]
Yeah, probably a github bug.

**Pull request merged**